dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -221,6 +222,35 @@ case class BinaryOutcomeDLCWithSelf(
     txBuilderF.flatMap(subtractFeeAndSign)
   }
 
+  /** Constructs the (time-locked) refund transaction for when the oracle disappears
+    * or signs an unknown message.
+    * Note that both parties have the same refund transaction.
+    */
+  def createRefundTx(
+      fundingSpendingInfo: MultiSignatureSpendingInfo): Future[Transaction] = {
+    val toLocalValueNotSat =
+      (fundingSpendingInfo.amount * localInput).satoshis.toLong / totalInput.satoshis.toLong
+    val toLocalValue = Satoshis(Int64(toLocalValueNotSat))
```

Comment:
Can you make a small PR with a constructor on `Satoshis` that just takes a scala `Long` so we don't always have to wrap in a `Int64()`? This is annoying and i've been meaning to do it forever..

Reply:
#899